src/components/atoms/Text/Text.tsx

```diff
@@ -53,13 +48,6 @@ const Text = styled.span<ITextProps>`
   letter-spacing: 0;
   color: ${({ color = colors.greyDarkest }) => color};
 
-  ${({ mb = false }) =>
-    mb &&
-    `
-    display: block;
-    margin-bottom:   24px
-  `};
```

Comment:
I wonder if we could add `display: block` by parsing `className`:
```js
${({ className, as }) => as === 'span' && className.split(' ').some(clss => clss.includes('mb')) && `
  display: block
`}
``` 
The reasoning being `<Text />` by default renders as a `<span />` and it won't be nice for clients to have to do wrapper components every time:
```js
const BlockText = styled(Text)`display: block`
<BlockText className="mb-6" />
```